packages/ra-supabase-core/src/useSupabaseAccessToken.ts

```diff
@@ -0,0 +1,33 @@
+import { useRedirect } from 'ra-core';
+import { useEffect } from 'react';
+
+/**
+ * This hook gets the access_token from supabase in the current browser URL and redirects to the specified page (/ by default) if there is none.
+ * To be used in pages such as those which set the password after a reset or an invitation.
+ **/
```

Comment:
could you please add a suage example in the jsdoc?

Reply:
Done